app/assets/stylesheets/partials/_autocomplete.scss

```diff
@@ -0,0 +1,12 @@
+// the autocomplete does not set a font, this only effects the autocomplete
+// items, so we set it here
+li.autocomplete__option {
+  @include govuk-font($size: 19);
```

Comment:
I think a problem remains with the fonts. I believe the option shown in the list below is correct but the 'selected' option is using something different. It's a close match but the bottom of the "l" character is different.

![Screenshot 2020-04-08 at 11 05 45](https://user-images.githubusercontent.com/912473/78772874-5236eb00-798a-11ea-8401-d008055e067d.png)

When I inspect the 'selected' element Firefox says that it's something other than GDS transport.

![Screenshot 2020-04-08 at 11 17 00](https://user-images.githubusercontent.com/912473/78773036-8ca08800-798a-11ea-9da8-ac8f0a5c51a7.png)

I'm surprised by this type of problem in a GDS package. It looks like their [demo page](https://alphagov.github.io/accessible-autocomplete/examples/) has the same issue though so I'm not sure it would be a quick fix.
![Screenshot 2020-04-08 at 11 19 40](https://user-images.githubusercontent.com/912473/78773404-16e8ec00-798b-11ea-8658-fc88b5b8964f.png)

Reply:
Wow! Great spot! I'll take a look - yeah not sure why it is like this.